location/tests.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+from django.urls import reverse
+from .models import Location
+import string
+
+
+class LocationModelTests(TestCase):
+    fixtures = ["locations.json"]
+
+    def test_create_model(self):
+        loc = Location.objects.create()
+        self.assertIsNotNone(loc)
+
+    def test_google_map_url(self):
+        loc = Location.objects.get(pk=1)
+        print(loc.google_map_url)
```

Comment:
is this `print()` statement still necessary or is it left over from debugging?

Reply:
Oops! I forgot to delete it. Thanks for pointing